pxtlib/emitter/emitter.ts

```diff
@@ -448,10 +448,38 @@ namespace ts.pxt {
             }, arg0, arg1, arg2));
         }
 
-        function unhandled(n: Node, addInfo = "") {
-            if (addInfo)
-                addInfo = " (" + addInfo + ")"
-            return userError(9202, lf("Unsupported syntax node: {0}", stringKind(n)) + addInfo);
+        function unhandled(n: Node, info?: string) {
+            //If we info then we may as well present that instead
+            if (info) {
+                return userError(9202, `Unsupported syntax: ${info}`)
```

Comment:
You must use localizable strings in error messages using the ``lf`` macro:

    lf("Unsupported syntax: {0}", info)